releaseNotes/JavaScriptProvider.md

```diff
@@ -0,0 +1,33 @@
+# Azure App Configuration JavaScript Provider
+
+[Source code][source_code] | [Package (npm)][package] | [Samples][samples]
+
+## 1.0.0-preview.2 - December 15, 2023
+## Breaking Changes
+- Changed the behavior when multiple labels are filtered in one selector. In the JavaScript provider configuration settings are loaded into a `Map` containing `key` and `value`, without preserving the `label`. If multiple labels are matched in one selector, the provider cannot determine which one to use. Now it will throw an error if label filter contains `*` or `,`. [#22](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/22)

```

Comment:
Will this be easier for customers to understand the impact?

The label filter in a selector is restricted to a single label. An error will be thrown if a label filter contains `*` or `,`. This change is to avoid the ambiguity when multiple values are loaded for the same key. Key-values with different labels can still be loaded using separate selectors for proper configuration composition.

BTW, is this the same behavior as the .NET provider (throwing error)? I thought we treated `*` as literal if it is present in a label filter.

Reply:
It’s consistent with dotnet. See:
https://github.com/Azure/AppConfiguration-DotnetProvider/blob/e9a3aa5b49958bd6190fb91c7a16d10ac46e3967/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs#L163

Reply:
Thanks for the confirmation, @Eskibear.